invary_runtime_integrity/manifest.json

```diff
@@ -0,0 +1,41 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "13509f2d-d922-4d8b-b3c2-7a8c2dd7fc54",
+  "app_id": "invary-runtime-integrity",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Visualize the Runtime Integrity of your operating systems",
+    "title": "Invary Runtime Integrity",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Invary Runtime Integrity OOTB Dashboard",
+        "image_url": "images/dashboard.png"
+      }
+    ],
+    "classifier_tags": [
+      "Category::Automation",
+      "Category::Log Collection",
+      "Category::OS & System",
+      "Category::Security",
+      "Submitted Data Type::Logs",
+      "Supported OS::Linux"
+    ]
+  },
+  "assets": {
+    "dashboards": {
+      "Invary Runtime Integrity": "assets/dashboards/invary_runtime_integrity.json"
+    }
```

Comment:
Apologies for missing this earlier, but you'll also need to add an `integration` object to this assets list. See [this integration](https://github.com/DataDog/integrations-extras/blob/master/ably/manifest.json) for an example.